yarn.lock

```diff
@@ -1213,33 +1213,33 @@
 
 "@devseed-ui/accordion@4.1.0":
   version "4.1.0"
-  resolved "http://3.220.121.64:4873/@devseed-ui%2faccordion/-/accordion-4.1.0.tgz#622483e69e553a4f1e8d4a8c44f61ad4cfa1395b"
+  resolved "http://verdaccio.ds.io:4873/@devseed-ui%2faccordion/-/accordion-4.1.0.tgz#622483e69e553a4f1e8d4a8c44f61ad4cfa1395b"
```

Comment:
@danielfdsilva What is Verdaccio? (I think you told me about this when I first started, but obviously, I forgot about it 😓 )

Reply:
Verdaccio is our private npm registry. We were mostly using it when we had beta versions of the ui library, and even though we currently don't have any private libraries, requests still go through it